include/vast/Dialect/Core/Func.td

```diff
@@ -22,9 +22,8 @@ class Core_FuncBaseOp< Dialect dialect, string mnemonic, list< Trait > traits =
   : Op< dialect, mnemonic,
     !listconcat(traits, [
       AutomaticAllocationScope,
-      CallableOpInterface,
-      FunctionOpInterface,
```

Comment:
Do we have to remove `CallableOpInterface` too?

Reply:
It is implied by `FunctionOpInterface`